game-core/src/main/java/games/strategy/triplea/delegate/battle/MustFightBattle.java

```diff
@@ -363,7 +365,29 @@ public BattleRound getBattleRoundState() {
   }
 
   @Override
-  public Collection<Unit> getUnits(final Side... sides) {
+  public GamePlayer getPlayer(final Side side) {
+    return side == OFFENSE ? getAttacker() : getDefender();
+  }
+
+  @Override
+  public Collection<Unit> getUnits(final UnitsStatus status, final Side... sides) {
```

Comment:
Do I understand right that we're doing more of a filter operation here rather than a straight get? If so, what would you think of naming this method 'filterUnits'?  Or 'getUnitsByStatus', or 'filterUnitsBySTatus'

Reply:
`filterUnits` sounds like it should take a collection of units and return a filtered subset.  This is more of a `get`.  I also don't think adding `ByStatus` to it is needed since the first parameter is the status so it already indicates you are getting by status.  But I could change it to `getUnitsByStatus`.